pkg/ip/iface.go

```diff
@@ -62,28 +64,36 @@ func GetInterfaceIP4Addr(iface *net.Interface) (net.IP, error) {
 		}
 
 		if addr.IP.IsGlobalUnicast() {
-			return addr.IP, nil
+			ipAddrs = append(ipAddrs, addr.IP)
+			continue
 		}
 
 		if addr.IP.IsLinkLocalUnicast() {
 			ll = addr.IP
```

Comment:
for 

should we also allow multi linl-local addrs ?

like

```go
	// prefer non link-local addr
	var ll []net.IP

	if addr.IP.IsLinkLocalUnicast() {
		ll = append(ll, addr.IP)
	}
```

since we append it to last, so it does not break our need " prefer non link-local addr"

Reply:
already changing the function and we need to doa minor release for this to go in so ya, why shouldnt we i guess is a better question

Reply:
OK, I push another commit, and also renamed the func name to add `s`